product-updates/knowledge-center/view-submission-history.html

```diff
@@ -308,30 +308,157 @@ <h3 class="usa-process-list__heading">Search for the reporting period to view</h
+
+    </tr>
+    <tr>
+      <th scope="row"><img src="../img/alerts/success.svg" alt="" aria-hidden="true" style="width: 20px; height: 20px; position: relative; top: 4px; margin-right: 5px;">Accepted</th>
+      <td>
+        Your file has been processed and contains no errors.
+      </td>
+
+    </tr>
+    <tr>
+      <th scope="row"><img src="../img/alerts/error.svg" alt="" aria-hidden="true" style="width: 20px; height: 20px; position: relative; top: 4px; margin-right: 5px;">Accepted with errors</th>
+      <td>
+        Your file was processed but some cases within your data contain errors.
+      </td>
+
+    </tr>
+    <tr>
+      <th scope="row"><img src="../img/alerts/error.svg" alt="" aria-hidden="true" style="width: 20px; height: 20px; position: relative; top: 4px; margin-right: 5px;">Partially accepted with errors</th>
+      <td>
+        Your file was unable to fully process. Cases and records within your data may contain errors.
+      </td>
+
+    </tr>
+       <tr>
+      <th scope="row"><img src="../img/alerts/cancel.svg" alt="" aria-hidden="true" style="width: 24px; height: 24px; position: relative; top: 6px; margin-right: 5px;">Rejected</th>
+      <td>
+        Your file could not be processed and likely contains errors relating to formatting and file structure.
+      </td>
+
+    </tr>
+  </tbody>
+</table>
+
+<table class="usa-table">
+  <caption>
+   Aggregate Data Columns Key
+  </caption>
+  <thead>
+    <tr>
+      <th scope="col">Column Name</th>
+      <th scope="col">Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th scope="row">Cases Without Errors</th>
+      <td>
+        The total number of TANF Cases in your file that do not have any errors associated with them.
+      </td>
+
+    </tr>
+    <tr>
+      <th scope="row">Cases With Errors</th>
+      <td>
+        The total number of TANF Cases in your file that have one or more error associated with them. This likely indicates problems with your data such as inconsistent values or using values outside those defined in the <a href="">TANF/SSP Coding Instructions</a> or <a href="">Tribal TANF Coding Instructions</a>.
+      </td>
+
+    </tr>
+    <tr>
+      <th scope="row">Records Unable To Process</th>
+      <td>
+        The total number of records within your file that the system was not able to "read" to check for errors. This likely indicates problems with the layout and coding of data within your file, e.g. A year value that's supposed to be formatted YYYY (ex. 2024) being entered as YY (ex. 24).  
+      </td>
+
+    </tr>
+    <tr>
+      <th scope="row">Total Errors</th>
+      <td>
+        Since section 3 and 4 data contains aggregate values in one record rather than TANF case data with many records, TDP can only provide a higher level error count. Errors here likely relate to inconsistencies between your Active and Closed Caseloads and your aggregate reporting or to issues with layout and coding like Records Unable to Process.
```

Comment:
```suggestion
        Since section 3 and 4 data contains aggregate values in one record rather than case data with many records, TDP can only provide a higher level error count. Errors here likely relate to values outside those defined in the <a href="">TANF/SSP Coding Instructions</a> or <a href="">Tribal TANF Coding Instructions</a> and/or inconsistencies in the values between related elements for a given reporting month. 
```